.github/workflows/frontend-test.yml

```
@@ -1,42 +1,51 @@
-            - name: Set up Node.js
-              uses: actions/setup-node@v3
-              with:
-                  node-version: '16.14'
-            - name: Install dev dependencies
-              run: npm install
-            - name: Run lint test
-              run: make lint
+ lint:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)